doc/release-notes.md

```diff
@@ -86,6 +86,10 @@ Changes to GUI or wallet related settings can be found in the GUI or Wallet sect
 New settings
 ------------
 
+- A new `mempoolfullrbf` option has been added, which enables the mempool to
+  accept transaction replacement without enforcing the opt-in replaceability
+  signal. (#25353)
```

Comment:
nit in 4c9666bd73645b94ae81be1faf7a0b8237dd6e99, Just in case we add another way of signaling replaceability...
```suggestion
  accept transaction replacement without enforcing BIP125 opt-in replaceability
  signal. (#25353)
```